.drone.yml

```
@@ -871,3 +871,31 @@ steps:
       api_server: https://api.staging.corp.mongodb.com
       kubernetes_token:
         from_secret: staging_kubernetes_token
+---
+kind: pipeline
+type: kubernetes
+name: publish-mongodb-chatbot-evaluation
+depends_on: ["test-all"]
+
+trigger:
+  event:
+    - tag
+  ref:
+    include:
+      - refs/tags/mongodb-chatbot-evaluation-v*
+
+steps:
+  - name: build package
+    image: node:18
+    commands:
+      - npm install lerna
```

```suggestion
      - npm install
```
We may add other dependencies (e.g. we just added `@stylistic/eslint-plugin-js`) should we default to installing everything?

---

that's covered in the following `npm run bootstrap` step. it runs install

---

though if we upgrade lerna to latest, i believe they deprecated `bootstrap`, so we could refactor then.